docusaurus.config.ts

```diff
@@ -89,7 +134,55 @@ const config: Config = {
             from: "/csv",
             to: "/apps/guides/csv-support",
           },
+          // Lockup redirects from old docs to new docs
+          {
+            from: "/concepts/sablier-protocol",
+            to: "/concepts/what-is-sablier",
+          },
+          {
+            from: "/concepts/protocol/streaming",
+            to: "/concepts/streaming",
+          },
+          {
+            from: "/concepts/protocol/nft",
+            to: "/concepts/nft",
+          },
+          {
+            from: "/concepts/protocol/fees",
+            to: "/concepts/fees",
+          },
+          {
+            from: "/concepts/protocol/transferability",
+            to: "/concepts/transferability",
+          },
+          {
+            from: "/contracts/v2/security",
+            to: "/concepts/security",
+          },
+          {
+            from: "/contracts/v2/deployments/v1.1",
+            to: "/guides/lockup/previous-deployments/v1.1",
+          },
+          {
+            from: "/contracts/v2/deployments/v1.0",
+            to: "/guides/lockup/previous-deployments/v1.0",
+          },
+          {
+            from: "/contracts/v1/deployments",
+            to: "/guides/legacy/deployments",
+          },
         ],
+        createRedirects(existingPath) {
+          if (existingPath.includes("/community")) {
```

Comment:
What do we need this `if` for?

Reply:
I copied the snippet from docusaurus docs and forgot to replace the community keyword. It should be protocol etc. Will change it in a while.

Reply:
https://github.com/sablier-labs/docs/pull/186/commits/960f13d57116de8ebf8585df4da2ab18e8317f79

Whether it works or not, will test it in production.